src/Backend/Job.hs

```diff
@@ -36,16 +36,15 @@ queueName = "restyled:restyler:jobs"
 
 processJob :: HasDB env => ExecRestyler (RIO env) -> Entity Job -> RIO env ()
 processJob execRestyler eJob@(Entity jobId job) = do
-    now <- liftIO getCurrentTime
```

Comment:
:man_facepalming: "let's just grab `now` before doing the thing, then use that to set `completedAt` later..."